modules/fedimint-ln-client/src/lib.rs

```diff
@@ -1006,15 +1032,15 @@ impl LightningClientModule {
         // it/bounds for it
         let fee = match &client_output.output {
             LightningOutputV0::Contract(contract) => {
+                let amt = amount
+                    .map(|a| a.msats)
+                    .or(invoice.amount_milli_satoshis())
+                    .ok_or(anyhow!("MissingInvoiceAmount"))?;
```

Comment:
I'd like to assert that `invoice.amount_milli_satoshis()` is `Some` no matter what.

Reply:
it was enforced implicity by either calling `create_incoming_output` or `create_outgoing_output` but will do so just to be sure